### PR TITLE
MI Hunting Data

### PR DESCRIPTION
**Pull request by FishheadNate**
- Created directory for MI
- Created json files for: Wildife Management Units, State Parks open for Hunting, State Forest/Wildlife/Game Areas open for hunting, Private Land open to Public Hunting, & Commerical Forests open for hunting.
- Removed the PA data that was pushed with this branch

### CODE REVIEW

File: sources/US/MI/state-areas-2017.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name" : "MI State Forest, Wildlife, and Game Areas Open to Hunting 2017",
+    "url": "http://gis-midnr.opendata.arcgis.com/datasets/403b88ca6cc443a59e54eb28e4f4de98_5#",
```
Comment (JesseCrocker) [inline sources/US/MI/state-areas-2017.json]: This url goes to a webpage, not a zip file

File: sources/US/MI/hunter-access-2017.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (JesseCrocker) [inline sources/US/MI/hunter-access-2017.json]: There's an error processing this file, but i dont think it's your fault.

File: sources/US/MI/state-areas-2017.json (hunk 1 above)
Comment (FishheadNate) [reply sources/US/MI/state-areas-2017.json]: Ok. the Geojson link would never load in qgis for me so I grabbed the shp file link, but it had some text after the .zip extension. 

I’ll upload the shp to The S3 bucket and pull a url from there

File: sources/US/MI/hunter-access-2017.json (hunk 2 above)
Comment (FishheadNate) [reply sources/US/MI/hunter-access-2017.json]: Do I need to change the source url?

Comment (JesseCrocker) [reply sources/US/MI/hunter-access-2017.json]: No, it’s some sort of error with the software that reads these files, I’ll fix it, but I can’t merge this until I do.

**Comment by FishheadNate** [comment]
Changed to source url for the `state-areas2017.json` to pull from S3 bucket

**Pull request merged**